README.md

```diff
@@ -55,3 +55,15 @@ quill.enableMathQuillFormulaAuthoring({
 The operators variable is an array of arrays. The outside array contains all of the different arrays which describe the operator buttons. The arrays inside of the main array consist of two values. The first value is the latex that gets rendered as the value on the button, and the second value is the latex that gets inserted into the MathQuill editor.
 
 <img src="./screenshots/custom-operator-buttons.png" width="450" alt="Demo screenshot with custom operator buttons">
+
+### Generating minified javascript files
+
+Generation of minified javascript for this project uses uglifyjs. To generate minified javascript, first make sure that uglifyjs is installed globally(you may have to run this command as root):
+
+```
+npm install uglify-js -g
```

Comment:
Could you add this as a devDependency to `package.json` so that everything is self-contained and all the tools can be installed with the usual `npm install`?

Reply:
Should be fixed now. The package is under the `devDependencies` in `package.json`, so all that you have to do is run `npm install` and then `npm run build`.  I also updated `readme.md` to reflect the update.